File: SeleniumJava/com.crm.seleniumM4/src/webpageVerification/RedBus.py
from selenium import webdriver
import time

driver = webdriver.Chrome()
expectedURL = "https://www.redbus.in/"

driver.get(expectedURL)
driver.maximize_window() 
actualURL = driver.current_url
driver.close() 

if(actualURL == expectedURL):
    print("Same same")
else:
    print("Not same")

File: SeleniumJava/com.crm.seleniumM4/src/webpageVerification/RedBus.java
package webpageVerification;

import org.openqa.selenium.chrome.ChromeDriver;

public class RedBus {
	public static void main(String[] args) {
		String expectedURL = "https://www.redbus.in/";
		ChromeDriver driver = new ChromeDriver(); 
		driver.get(expectedURL);
		driver.manage().window().maximize();
		
		String actualURL = driver.getCurrentUrl(); 
		System.out.println(expectedURL.equals(actualURL) ? "Same Same" : "Not same");

		driver.close();	
		
	}
}
